Classes/Property/TypeConverter/ExposedValueObjectConverter.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Netlogix\JsonApiOrg\AnnotationGenerics\Property\TypeConverter;
+
+use Neos\Flow\Property\PropertyMappingConfigurationInterface;
+use Neos\Flow\Property\TypeConverter\AbstractTypeConverter;
+use Netlogix\JsonApiOrg\AnnotationGenerics\Cache\ExposedValueObjectCache;
+use Netlogix\JsonApiOrg\AnnotationGenerics\Domain\Model\ExposedValueObjectInterface;
+
+class ExposedValueObjectConverter extends AbstractTypeConverter
+{
+
+    protected $sourceTypes = ['array', 'string'];
+
+    /**
+     * The target type this converter can convert to.
+     */
+    protected $targetType = ExposedValueObjectInterface::class;
+
+    public function convertFrom(
+        $source,
+        $targetType,
+        array $convertedChildProperties = [],
+        PropertyMappingConfigurationInterface $configuration = null
+    ): ?ExposedValueObjectInterface {
+        return ExposedValueObjectCache::get($source, $targetType);
```

Comment:
`$source` can be of type array because of Line 13. In that case, the identifier will be in array key `__identity`

```
if (is_arrray($source) && isset($source['__identity'])) {
  // get by $source['__identity']
}
```